test/optimizer/stats_storage_test.cpp

```diff
@@ -108,14 +110,18 @@ TEST_F(StatsStorageTests, InsertAndGetTableStatsTest) {
                                           table_stats_collector.get());
 
   VerifyAndPrintColumnStats(data_table.get(), 4);
```

Comment:
If I understand correctly, this function VerifyAndPrint... does print but very little verify. Acknowledge that this wasn't modified, but it does mean some of the old tests are not particularly useful.